Doc/library/argparse.rst

```diff
@@ -1930,7 +1930,26 @@ Mutual exclusion
 
    Note that currently mutually exclusive argument groups do not support the
    *title* and *description* arguments of
-   :meth:`~ArgumentParser.add_argument_group`.
+   :meth:`~ArgumentParser.add_argument_group`. However, you can still add a
+   mutually exclusive group to another group so they are organized within
+   the help messages as follows:
```

Comment:
If you look at L1921, it has two colons to mark a [source code block](https://devguide.python.org/documentation/markup/#source-code), so I'd include that here too.

The other thing is that you're looking to merge into the `python3.10` branch, but instead it should go directly into the `main` branch. I believe there's no way to change the target branch, so you'll need to close this PR and open a new one.

Reply:
Thank you! I'll do that.